keyboards/dotlen/queue/readme.md

```diff
@@ -0,0 +1,28 @@
+# Queue.len()
+
+![Queue.len()](https://i.imgur.com/8JKbm6Ph.png)
+
+my 65% keyboard with multiple layout support
```

Comment:
Missing layout support from the firmware?

Reply:
good call. The PCB has layout support, but I didn't write out all  of the LAYOUTs.  I renamed the layout in keyboard.json to be LAYOUT_all as  suggested in PR Checklist. Can I come  back with additional layouts, or just take out the comment altogether?

Reply:
never mind - I added two of the supported layouts for the keyboard.  Let me know if you were looking for something different